Gruntfile.js

```diff
@@ -1,9 +1,11 @@
 /**
  * @author Adam Meadows [@job13er](https://github.com/job13er)
- * @copyright 2015 Cyan, Inc. All rights reserved.
+ * @copyright 2015 Ciena Corporation. All rights reserved.
 */
 
-var helper = require('./src/grunt/helper');
+'use strict';
```

Comment:
I've intentionally brought it back. The `.eslintrc` is not wrong. The message we were getting about not needing it was for es6 modules because they are strict by default. This is not an es6 module, it's a CommonJS module, which, as far as I know, still needs the `'use strict'`;